x86/CheckTime.asm

```diff
@@ -1,86 +1,104 @@
-         calign   16
```

Comment:
An important function like CheckTime should be aligned to at least 16. Try keeping the align 16 or even upgrading it to align 64 and test the timings.